easybuild/easyconfigs/m/MitoHiFi/MitoHiFi-3.2-foss-2022b.eb

```diff
@@ -0,0 +1,77 @@
+easyblock = 'PythonBundle'
+
+name = 'MitoHiFi'
+version = '3.2'
+
+homepage = 'https://github.com/marcelauliano/MitoHiFi'
+description = "MitoHiFi is a Python workflow that assembles mitogenomes from Pacbio HiFi reads"
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+
+dependencies = [
+    ('Python', '3.10.8'),
+    ('SciPy-bundle', '2023.02'),  # for pandas
+    ('SAMtools', '1.17'),
+    ('CD-HIT', '4.8.1'),
+    ('minimap2', '2.26'),
+    ('hifiasm', '0.19.7'),
+    ('MAFFT', '7.505', '-with-extensions'),
+    ('Biopython', '1.81'),
+    ('matplotlib', '3.7.0'),
+    ('BEDTools', '2.30.0'),
+    ('Pillow', '9.4.0'),
+    ('bcbio-gff', '0.7.0'),
+    # ('MitoFinder', '1.4.1'),
```

Comment:
@deniskristak Isn't MitoFinder a required depenency for MitoHifi?

See for example https://github.com/marcelauliano/MitoHiFi/blob/v3.2/src/rotation.py#L76 and https://github.com/marcelauliano/MitoHiFi/blob/v3.2/src/parallel_annotation.py#L46

It seems like [MITOS](https://pypi.org/project/mitos) can be used instead if providing MitoFinder is not possible, but then the `--mitos` flag must be used (and `MITOS` must be available as dependency).

It possible these dependencies are not checked via `pip check`, since they're command line tools.

Reply:
specified Mitos flag as a default